templates/smaily-woocommerce-admin.php

```diff
@@ -452,6 +445,127 @@ class ="smaily-toggle"
 				</tbody>
 			</table>
 		</div>
+		<div id="rss">
+			<table class="form-table">
+				<tbody>
+					<tr class="form-field">
+						<th scope="row">
+							<label>
+								<?php echo esc_html__( 'Product limit', 'smaily' ); ?>
+							</label>
+						</th>
+						<td>
+							<input
+								type="number"
+								id="rss-limit"
+								name="rss_limit"
+								class="smaily-rss-options"
+								min="1" max="250"
+								value="<?php echo esc_html( $rss_limit ); ?>"
+							/>
+							<small>
+								<?php
+								echo esc_html__(
+									'Limit how many products you will add to your field. Maximum 250.',
+									'smaily'
+								);
+								?>
+							</small>
+						</td>
+					</tr>
+					<tr class="form-field">
+						<th scope="row">
+							<label for="rss-category">
+								<?php echo esc_html__( 'Product category', 'smaily' ); ?>
+							</label>
+						</th>
+						<td>
+							<select id="rss-category" name="rss_category" class="smaily-rss-options">
+								<?php
+								// Display available WooCommerce product categories and saved category.
+								foreach ( $wc_categories_list as $category ) : ?>
+									<option
+										value="<?php echo esc_html( $category->slug ); ?>"
+										<?php echo $rss_category === $category->slug ? 'selected' : ''; ?>
+									>
+										<?php echo esc_html( $category->name ); ?>
+									</option>
+								<?php endforeach; ?>
+								<option value="" <?php echo empty( $rss_category ) ? 'selected' : ''; ?>>
+									<?php echo esc_html__( 'All products', 'smaily' ); ?>
+								</option>
+							</select>
+							<small>
+								<?php
+								echo esc_html__(
+									'Show products from specific category',
+									'smaily'
+								);
+								?>
+							</small>
+						</td>
+					</tr>
+					<tr class="form-field">
+						<th scope="row">
+							<label for="rss_order_by">
+								<?php echo esc_html__( 'Order products by', 'smaily' ); ?>
+							</label>
+						</th>
+						<td id="smaily_rss_order_options">
+							<select id="rss-order-by" name="rss_order_by" class="smaily-rss-options">
+								<?php
+								$sort_categories_available = array(
+									'modified' => __( 'Modified At', 'smaily' ),
+									'none'     => __( 'None', 'smaily' ),
+									'id'       => __( 'ID', 'smaily' ),
+									'name'     => __( 'Name', 'smaily' ),
+									'type'     => __( 'Type', 'smaily' ),
+									'rand'     => __( 'Random', 'smaily' ),
+									'date'     => __( 'Created At', 'smaily' ),
+								);
```

Comment:
I don't think there is any real value for `none` sort order, and I would remove it. Plus I would keep the options in an alphabetical order even if default selected option is not first. It just makes finding options easier.

Reply:
Added